packages/core/src/bags/redux/middlewares/getOperationsOnBagRequestSuccess.js

```diff
@@ -0,0 +1,54 @@
+import {
+  ADD_ITEM_TO_BAG_SUCCESS,
+  DELETE_BAG_ITEM_SUCCESS,
+  GET_BAG_SUCCESS,
+  UPDATE_BAG_ITEM_SUCCESS,
+} from '../actionTypes';
+import {
+  doGetBagOperation,
+  resetBagOperationsEntities,
+  resetState,
+} from '../actions';
+import { getBag, getBagId } from '../selectors';
+import { getBagOperation } from '../../client';
+
+/**
+ * Middleware to fetch bag operation if a bag request is successful.
```

Comment:
```suggestion
 * Middleware to clear previous operations and to fetch the operation of the newly executed bag request, if successful.
```

I'd just add something explaining that the operations are reset first, something like the suggestion 🙂

Reply:
Done!